testsuite/tests/flambda/incompatible_arrays.ml

```diff
@@ -1,5 +1,6 @@
 (* TEST
  flambda2;
+ arch_amd64;
```

Comment:
Tests for supported layouts should ideally still run - can you add CRs to re-enable the files that aren't specific to float32/vec128?

Reply:
The particular test you commented on only makes sense with `float32#`, but I've added CRs to others.